bin/PullAndProcessGsAnalysisData.sh

```diff
@@ -257,27 +292,30 @@ function mergeSamplesheets(){
 	local _captkit
 	for i in "${uniqProjects[@]}"
 	do
+		sampie="${TMP_ROOT_DIR}/${_batch}/${i}.csv"
+		cp "${sampie}"{,.converted}
+		printf '\n'     >> "${sampie}.converted"
+		sed -i 's/\r/\n/g' "${sampie}.converted"
+		sed -i "/^[\s${SAMPLESHEET_SEP}]*$/d" "${sampie}.converted"
+		mv "${sampie}.converted" "${sampie}"
+		
 		if [[ "${teller}" -eq '0' ]]
 		then
 			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Creating new combined samplesheet for the processing of the Analysis data"
 			#Renaming samplesheet to solely the GS_XX (e.g. GS_182.. so without the A,B,C etc suffix)
 			_projectName=$(echo "${i}" | grep -Eo 'GS_[0-9]+')
 			_captkit=$(echo "${i}" | awk 'BEGIN {FS="-"}{print $NF}')
 			_projectName="${_projectName}-${_captkit}"
-			# create a combined samplesheet header, renamed project to originalproject and added new project column
 			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "new samplesheet name: ${_projectName}.csv"
-			head -1 "${TMP_ROOT_DIR}/Samplesheets/${i}.csv" | perl -p -e 's|,project|,originalproject|' | awk '{print $0",project,gsBatch"}' > "${TMP_ROOT_DIR}/${_batch}/${_projectName}.csv"
-			tail -n+2 "${TMP_ROOT_DIR}/Samplesheets/${i}.csv" >> "${TMP_ROOT_DIR}/${_batch}/${_projectName}.csv"
+			cat "${TMP_ROOT_DIR}/${_batch}/${i}.csv" > "${TMP_ROOT_DIR}/${_batch}/${_projectName}.csv"
```

Comment:
Useless use of `cat`; a simple `cp` will do the job.

Reply:
fixed